stetho/src/main/java/com/facebook/stetho/inspector/database/DatabasePeerManager.java

```diff
@@ -129,7 +187,13 @@ private SQLiteDatabase openDatabase(String databaseName) throws SQLiteException
   }
 
   public interface ExecuteResultHandler<T> {
-    public T handleResult(Cursor result) throws SQLiteException;
+    public T handleRawQuery(Cursor result) throws SQLiteException;
```

Comment:
This shouldn't be `Cursor`; it implies that the cursor came from `database.rawQuery`.  Instead, prefer to parse `changes(), last_insert_row_id()` into two longs (or whatever type they are?)